scripts/microbe_stage/biome.as

```diff
@@ -1,5 +1,5 @@
 // Global table which stores the current biome the player is in.
-uint64 currentBiome = 0;
+uint64 currentBiome = 42;
```

Comment:
This should be propably set to something like `1000000` to make sure that we won't have problems if we add a bunch of more biomes.